pkg/api/filters/filter.go

```diff
@@ -0,0 +1,40 @@
+package filters
```

Comment:
Any thoughts about renaming this package to `filter`? There's no hard and fast rule we've observed on package naming so far, but I'm just thinking that `filter.Chain` reads better than `filters.Chain`.

Reply:
Yeah, agree. I forgot to rename it before i submitted the review :)